lib/customer.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'menu'
+
+class Customer
+
+	attr_accessor :customer_order
+
+	def initialize
+		@customer_order = []
+	end
+
+
+ 	def place_order (choice)
```

Comment:
Inconsistent indentation detected.<br>Do not put a space between a method name and the opening parenthesis.<br>Tab detected.